lib/saluki-core/src/topology/graph.rs

```diff
@@ -1,14 +1,22 @@
-use std::collections::{HashMap, HashSet, VecDeque};
+#![allow(dead_code)]
```

Comment:
Is it common to use this directive?

Reply:
Not very common at all.

I'm only using it here to make the linters pass because we aren't using this code anywhere else yet, but we know we're going to and so the "you have dead code!!!" warning is real but not _relevant_.

I'll end up removing it in the immediate future once I continue wiring things up.